jest.config.js

```diff
@@ -13,6 +13,9 @@ module.exports = {
       lines: 100,
       statements: 100,
     },
+    "./src/utils.ts": {
```

Comment:
You could use `/* istanbul ignore */` to bypass those unnecessary coverage instead.